File: Application/Dto/Shared/ContactDto.cs
using System.Collections.Generic;

namespace Application.Dto.Shared
{
    public class ContactDto
    {
        public int Id { get; set; }
        public string Email { get; set; }
        public string Site { get; set; }
        public string Cel { get; set; }
        public string Zap { get; set; }
        public string Landline { get; set; }
        public List<SocialNetworkDto> socialnetworks { get; set; }

    }
}

File: Views/src/shared/components/contact/dtos/contact-dto.ts
import { SocialMediasDto } from "./social-medias-dto";


export class ContactDto {
  id!: number;
  email!: string;
  site!: string;
  cel!: string;
  zap!: string;
  landline!: string;
  socialMedias!: SocialMediasDto[];
}
